lib/index.js

```diff
@@ -33,7 +33,9 @@ function swaggerize(server, options) {
  * @returns {onmount}
  */
 function mount(server, options) {
-    Object.defineProperty(server, 'swagger', {
+    if(!options.version)
+        options.version = 'v1';
+    Object.defineProperty(server, 'swagger'+options.version, {
```

Comment:
I'm curious why Object.defineProperty is used in the first place.  By default, that's immutable, which prevents us from dealing with the `server.swagger` property outside of swaggerize-restify.  Also, it doesn't even look like the `server.swagger` proeprty is used anywhere?  Can @bardzusny provide some clarification here?